lib/constructs/operation.js

```diff
@@ -13,6 +13,7 @@ class Operation {
     this.idls = [idl];
     this.name = idl.name;
     this.static = idl.static;
+    this.generic = idl.idlType.generic;
```

Comment:
This assumes that the return type of all potential overloads are the same. (That's why `this.idls` is an array – all overloaded operations with the same name are grouped into the same `Operation` object.)